tests/unit/s2n_cipher_suite_match_test.c

```diff
@@ -486,7 +486,7 @@ int main(int argc, char **argv)
          * configured.
          */
         {
-            const struct s2n_cipher_suite *expected_wire_choice = &s2n_rsa_with_rc4_128_md5;
+            const struct s2n_cipher_suite *expected_wire_choice = &s2n_rsa_with_3des_ede_cbc_sha;
```

Comment:
Wait this actually doesn't look right. We should choose the first available RSA cipher suite. If RC4 is available, that should be RC4. Are you sure your feature probe ever reports that RC4 is available?

Reply:
Ah, you added it to cmake, but not to make, which is apparently still hanging around in the CI running the unit tests -_- https://github.com/aws/s2n-tls/blob/main/s2n.mk#L216-L220